packages/puppeteer-core/src/api/HTTPResponse.ts

```diff
@@ -157,7 +157,7 @@ export class HTTPResponse {
    * True if the response was served by a service worker.
    */
   fromServiceWorker(): boolean {
```

Comment:
should we mark it as abstract to make sure we don't accidentially start returning false for everything?

Reply:
Happy to refactor this in another PR as this class is not abstract. Only moving it into the BiDi class for now.